testng/src/main/java/cucumber/api/testng/CucumberFeatureWrapperImpl.java

```diff
@@ -22,6 +22,6 @@ public CucumberFeature getCucumberFeature() {
 
     @Override
     public String toString() {
-        return cucumberFeature.getGherkinFeature().getFeature().getName();
+        return cucumberFeature.getPath();
```

Comment:
I think it still should be the feature name here (possibly surrounded by "") for the sake of consistency, we have tried to standardise on the feature name as the "classname" see #1167.